private/pkg/protosource/file.go

```diff
@@ -143,7 +143,9 @@ func (f *file) PyGenericServices() bool {
 }
 
 func (f *file) PhpGenericServices() bool {
-	return f.fileDescriptor.GetOptions().GetPhpGenericServices()
+	// Support for PhpGenericServices was removed in
```

Comment:
This isn't really the "right" fix, really  we should be removing all of this. Give me a minute.